packages/common/test/discussions/edit.test.ts

```diff
@@ -70,6 +72,12 @@ describe("discussions edit:", () => {
           return Promise.resolve(newModel);
         }
       );
+      const createSettingsSpy = spyOn(
```

Comment:
`createSetting` returns a `Promise<IEntitySetting>` but this spy returns a `Partial<IEntitySetting>`. Additionally, `createSettingsSpy` is an unused const in the current test state. We should:

1. Update the spy to return Promise<IEntitySetting>` (has `id` property that matches the item id)
2. Check that `createSettingsSpy` is called with what we'd expect it to be called with.
3. Verify the values of `chk.entitySettingsId` and `chk.discussionSettings` are the values we'd expect